cli/app.go

```diff
@@ -311,6 +317,16 @@ func parseStructFromCtx[T any](ctx *cli.Context) T {
 	return t
 }
 
+func getGlobalArgs(ctx *cli.Context) (*globalArgs, error) {
+	gArgs := parseStructFromCtx[globalArgs](ctx)
+	// TODO(RSDK-9361) - we shouldn't let people create global args directly, to enforce this
```

Comment:
> to enforce this

Piece of comment missing here? Are you enforcing anything now? Or saying we should `TODO` that enforcing?

Reply:
Ah yeah, good question! The "this" in question is the "you can't set both a profile and the disable-profiles flag" logic, which isn't enforced at all when a user creates global args directly. Updated the comment to make it more clear :)